onnx_converter/README.md

```diff
@@ -1,12 +1,10 @@
 # HugeCTR to ONNX Converter #
-hugectr2onnx is a python package that can convert HugeCTR models to ONNX format. It can improve the compatibility of HugeCTR with other deep learning frameworks given that Open Neural Network Exchange (ONNX) serves as an open-source format for AI models.
 
-To use the ONNX converter, you need to prepare HugeCTR models files and graph configuration files which can be obtained with HugeCTR training APIs. Please refer to [HugeCTR Python Interface](../docs/python_interface.md) to get familiar with how to train and save your models. To learn the workflow of training HugeCTR model, converting to ONNX and making inference with ONNX Runtime, please refer to [HugeCTR to ONNX Demo Notebook](../notebooks/hugectr2onnx_demo.ipynb).
+The HugeCTR to Open Neural Network Exchange (ONNX) converter is a `hugectr2onnx` python package that can convert HugeCTR models to ONNX. It can improve the compatibility of HugeCTR with other deep learning frameworks since ONNX serves as an open-source format for AI models.
```

Comment:
We may want to reword it as "The HugeCTR to Open Neural Network Exchange (ONNX) converter `hugectr2onnx` is a  python package that ...".